weave-js/src/components/Form/AutoComplete.tsx

```diff
@@ -63,6 +63,10 @@ const getStyles = (props: AdditionalProps) => {
                 minHeight: `${HEIGHTS[size]} !important`,
               },
             },
+            '& .MuiAutocomplete-popupIndicator': {
+              borderRadius: '4px',
+              padding: '4px',
```

Comment:
Adding some styling for chevron menu open indicator, 
![Screenshot 2024-11-04 at 3 46 41 PM](https://github.com/user-attachments/assets/529f6d42-87a4-4c9e-8a22-d27d45a72892)